server/src/immich/controllers/person.controller.ts

```diff
@@ -26,16 +27,19 @@ export class PersonController {
   constructor(private service: PersonService) {}
 
   @Get()
-  getAllPeople(@AuthUser() authUser: AuthUserDto): Promise<PersonResponseDto[]> {
-    return this.service.getAll(authUser);
+  getAllPeople(
+    @AuthUser() authUser: AuthUserDto,
+    @Query('areHidden') areHidden: boolean,
```

Comment:
We use withDeleted, with thumbnails, etc. Maybe withHidden could be used instead.